README.md

```diff
@@ -133,6 +134,28 @@ data:
   password: MTIzNA==
 ```
 
+The controller needs to access AWS Secrets Manager via a role. How depends on your cluster setup, [kube2iam](https://github.com/jtblin/kube2iam) and [kiam](https://github.com/uswitch/kiam) are the most common IAM role plugins.
```

Comment:
Maybe I am wrong, but plugins are not necessary when your cluster runs within AWS. Eg, https://docs.aws.amazon.com/eks/latest/userguide/service_IAM_role.html

Can you clarify this?

Reply:
Some EKS users use kube2iam or another project to implement IAM roles for `Pods`.  We expect native EKS support soon, so we haven't bothered to run kube2iam or one of the other projects.

Reply:
Since you've published this as open source, I wouldn't assume that everyone is running EKS(wasn't clear to me either from this project description/doc/readme); some are running their own clusters on top of EC2 in which case you'd need this. I could rephrase it. Wdyt?

Reply:
This is AWS specific and I'd keep this out of the project documentation.